app/utils/addressBookHelpers.ts

```diff
@@ -0,0 +1,17 @@
+import { AddressBookEntry } from './types';
+import { findEntries } from '~/database/AddressBookDao';
+
+export async function lookupAddressBookEntry(
+    address: string
+): Promise<AddressBookEntry | undefined> {
+    const entries = await findEntries({ address });
+    return entries[0];
+}
+
+/**
+ * Attempts to find the address in the accounts, and then AddressBookEntries
```

Comment:
As far as I can see this only checks something in the address book table, and does not check anything in the accounts table.

Reply:
Yes, this comment was way outdated.